lib/govuk_publishing_components/presenters/button_helper.rb

```diff
@@ -77,6 +79,7 @@ def css_classes
         css_classes << "govuk-button--start" if start
         css_classes << "gem-c-button--secondary" if secondary
         css_classes << "gem-c-button--secondary-quiet" if secondary_quiet
+        css_classes << "govuk-button--secondary" if secondary_solid
```

Comment:
Sorry if I'm being dense, but where is the new style being added? Won't this just add the existing green text version?

Reply:
This comes from the Design System (`govuk-frontend` via npm): https://github.com/alphagov/govuk-frontend/blob/cf5be781776d7bb24394dd9211f672588ae081f6/src/govuk/components/button/_index.scss#L172-L202